### ISSUE TITLE
Problem with : (colon) inside text

### ISSUE THREAD

**Issue by burgerboydaddy** [i18next/i18next-parser#67]
Hello,
I'm trying to parse text with : (colon) inside it.
For example: "Hello, I have: one, two, three"
Parser will separate this into 2 strings. "Hello, I have" and "one, two, three". 
Any suggestion how to handle this case (without splitting into separated strings)?
Thanks

**Comment by karellm** [comment]
It looks like a bug. Can you make a PR with a failing test case? Thanks!

**Comment by opposite-bracket** [comment]
@burgerboydaddy I had the same issue but this can be solved by defining a different `namespaceSeparator`. Here's an example of what my function call in gulp looks like:

```
gulp.task('i18next:parser', function () {
...
      .pipe(i18nextParser({
          ...
          namespaceSeparator: '__SS',
          ...
      }))
...
});
```

Of ocurse, we're not using namespaces but it is up to you to configure it as you want it. Hope that helps.

**Comment by karellm** [comment]
This will also be addressed in `1.0.0`. You can follow progress in #72. It should be ready with a week or two.

**Comment by karellm** [comment]
I second what @opposite-bracket suggested. After looking into it, this is not a bug but a feature. If you rely on keys that are sentences and might use `:`, you should disable or change the `namespaceSeparator` option.
